js/web/lib/wasm/jsep/webgpu/ops/common.ts

```diff
@@ -776,10 +776,10 @@ class ShaderHelperImpl implements ShaderHelper {
     @builtin(workgroup_id) workgroup_id : vec3<u32>,
     @builtin(num_workgroups) num_workgroups : vec3<u32>`;
     const globalIdxDefinition = is1DimensionDispatch ?
-        'let global_idx = global_id.x;' :
+        'let global_idx = global_id.x; let local_idx = local_id.x;' :
```

Comment:
if `local_idx` always equals to the built-in `local_invocation_index`, we can simply always keep in in the parameter list:
`@builtin(local_invocation_index) local_idx: u32,`